SportsNews/Helpers/UserInfoHelper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Identity;
+using SportsNews.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SportsNews
+{
+    public static class UserInfoHelper
+    {
+
+        public static string GetUserImage(ApplicationDbContext applicationDbContext, ClaimsPrincipal user)  
+        {
+            if (user.Identity.IsAuthenticated)
+            {
+                var userId = applicationDbContext.Users.FirstOrDefault(u => u.UserName == user.Identity.Name)?.Id ?? String.Empty;
+                var picture = applicationDbContext.UserPhotos.FirstOrDefault(u => u.UserId == userId)?.ProfilePicture ?? String.Empty;
```

Comment:
Let's implement UoW + Repository pattern for data access. 
Remove UserInfoHelper 
Implement UnitOfWork and Repository
https://docs.microsoft.com/en-us/aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-the-repository-and-unit-of-work-patterns-in-an-asp-net-mvc-application